products/views.py

```diff
@@ -27,4 +27,37 @@ def get(self, request, productgroup_id):
         ]
         mls    = [result['ml'] for result in results]
         prices = [result['price'] for result in results]
-        return JsonResponse({'products': results, 'mls': mls, 'prices': prices}, status=200)
\ No newline at end of file
+        return JsonResponse({'products': results, 'mls': mls, 'prices': prices}, status=200)
+
+
+class ProductListView(View):
+  def get(self,request):
+    try:
+      category_subcategory_id = request.GET.get('category_subcategory_id', None)
+      type_ml                 = request.GET.get('ml', None)
+      image                   = Image.objects.all()
+      
+      filter_set = {
+        'categorysubcategory_id': category_subcategory_id,
+        'ml'                    : type_ml
+      }
+      
+      products = Product.objects.filter(**filter_set) 
+
+      products = [{   
+        'id'             : product.id,
+        'name'           : product.name,
+        'ml'             : product.ml,
+        'price'          : product.price,
+        'productgroup_id': product.productgroup.id,
+        'image'          : [img for img in image.filter(product_id = product.id).values('image_url')],
```

Comment:
image를 불러올 때, Image.objects.all()이 아니라 product. 부터 시작해서 가져올 수 있도록 수정해주세요~

```suggestion
        'images' : [ for image in product....]
```